src/midi/midi_fluidsynth.cpp

```diff
@@ -143,7 +143,7 @@ std::tuple<std::string, int> parse_soundfont_pref(const std::string& line)
 	return {filename, percent};
 }
 
-#if defined(WIN32)
+	#if defined(WIN32)
```

Comment:
More weird indentation... Not going to comment on every instance of this.

Reply:
Mass fixed it all 👍🏻